packages/client/src/chat.ts

```diff
@@ -699,3 +710,17 @@ export function updatePeopleTyping(): void {
   chat1.html(msg);
   chat2.html(msg);
 }
+
+function getChatMessage(msg: string, type: SelfChatMessageType): string {
+  let message = msg;
+  switch (type) {
+    case SelfChatMessageType.info:
+      message = `<span class="green">${message}</span>`;
+      break;
+    case SelfChatMessageType.error:
+      message = `<span class="red">${message}</span>`;
+      break;
+    default:
```

Comment:
use ensure all cases

Reply:
in default there's no message change but code could reach that point; should I add a simple break?

Reply:
ill add it

Reply:
the point of the function is to return values, not modify variables, get rid of the let